eng/scripts/appium-install.ps1

```diff
@@ -40,27 +40,49 @@ Find the script for that on the DevDiv Azure DevOps instance, Engineering team,
 
 param
 (
-    [string] $appiumVersion = '2.1.1',
-    [string] $windowsDriverVersion = '2.10.1',
-    [string] $androidDriverVersion = '2.29.4',
-    [string] $iOSDriverVersion = '5.2.0',
-    [string] $macDriverVersion = '1.7.2'
+    [string] $appiumVersion = '2.5.4',
+    [string] $windowsDriverVersion = '2.12.23',
+    [string] $androidDriverVersion = '3.5.1',
+    [string] $iOSDriverVersion = '7.16.1',
+    [string] $macDriverVersion = '1.17.3'
 )
 
 Write-Output  "Welcome to the Appium installer"
 
 Write-Output  "Node version"
 node -v
 
-Write-Output  "Installing appium $appiumVersion"
-npm install -g appium@$appiumVersion
+# If there's a ~/.appium folder, remove it as it can cause issues from v1
+# it might also generally have caching issues for us with our runs
+$appiumUserData = "$env:USERPROFILE/.appium"
+if (Test-Path $appiumUserData) {
+    Write-Output  "Removing $appiumUserData"
+    Remove-Item -Path $appiumUserData -Force -Recurse
+}
```

Comment:
Something similar happens here: https://github.com/dotnet/maui/blob/main/eng/pipelines/common/ui-tests-steps.yml#L81-L86

We might want to consolidate?

Reply:
Ended up getting rid of this entirely and also removed the steps in the yml since we are now managing our own local folder for the driver installs.